### PR TITLE
Added core tests for Matrix, Ellipse, Circle and TilingSprite

### PR DESCRIPTION
**Pull request by JessicaNeary**
Added matrix tests for prepend() and identity() functions.

Added contains if-branch tests on Ellipse and Circle shapes for 100% coverage. 

Added TilingSprite test for containsPoint true and false paths.

Added TilingSprite test for height and width getters and setters.

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
Thanks for the tests @JessicaNeary! We love more unit tests. 

Looks like there's a [small linting error](https://travis-ci.org/pixijs/pixi.js/builds/198704326#L305-L306), if you could fix that.

File: test/core/Matrix.js [hunk 1]
```diff
@@ -107,10 +107,60 @@ describe('PIXI.Matrix', function ()
...
         expect(PIXI.Matrix.IDENTITY instanceof PIXI.Matrix).to.be.true;
         expect(PIXI.Matrix.TEMP_MATRIX instanceof PIXI.Matrix).to.be.true;
     });
-});
 
+    it('should reset matrix to default when identity() is called', function ()
+    {
+        const matrix = new PIXI.Matrix();
+
+        matrix.set(2, 2, 2, 2, 100, 100);
```
Comment (bigtimebuddy) [inline test/core/Matrix.js]: Would it be possible to select non-repeating, non-default values here? Will help disambiguate and test the parameters vs properties. For instance:
```js
matrix.set(2, 3, 4, 5, 100, 200);

expect(matrix.a).to.equal(2);
expect(matrix.b).to.equal(3);
expect(matrix.c).to.equal(4);
expect(matrix.d).to.equal(5);
expect(matrix.tx).to.equal(100);
expect(matrix.ty).to.equal(200);
```

File: test/core/Matrix.js [hunk 2]
```diff
@@ -107,10 +107,60 @@ describe('PIXI.Matrix', function ()
         expect(m1.ty).to.equal(m2.ty);
     });
 
+    it('should prepend matrix', function ()
+    {
+        const m1 = new PIXI.Matrix();
+        const m2 = new PIXI.Matrix();
+
+        m2.set(2, 3, 4, 5, 100, 100);
```
Comment (bigtimebuddy) [inline test/core/Matrix.js]: I'd suggest setting `ty` to `100` or something not the same as `tx` to make the assert more useful for checking the parameters vs properties.

**Comment by bigtimebuddy** [comment]
Ping @JessicaNeary

**Comment by staff0rd** [comment]
Ping @JessicaNeary

**Pull request merged**